project_resource_calendar/views/equipment_calendar_view.xml

```diff
@@ -39,6 +39,14 @@
                            class="oe_inline"
                            invisible="1" attrs="{'readonly': [('is_task_event','=',True)]}"/>
                 </h2>
+
```

Comment:
the sector field must be added in the project_event/views/calendar_event_category_view.xml

Reply:
@rimbendhaou Done